test/bin/test-citgm.js

```diff
@@ -2,13 +2,17 @@
 
 var test = require('tap').test;
 
-var spawn = require('../../lib/spawn');
+var spawn = require('child_process').spawn;
```

Comment:
@bzoz, I have tried running this pr on windows and it does indeed fix some of the issues but there are also other key changes that need to be made before you can actually run `npm test` on windows. Perhaps it's worth putting together a separate PR to get npm test fully running (and passing) on windows. I have managed to fix a few of the problems myself so we could always work on this together